applications/RANSApplication/rans_application_variables.h

```diff
@@ -77,6 +79,11 @@ namespace Kratos
     KRATOS_DEFINE_APPLICATION_VARIABLE( RANS_APPLICATION, int, RANS_IS_WALL_FUNCTION_ACTIVE )
     KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS( RANS_APPLICATION, FRICTION_VELOCITY )
 
+    // formulation specific variables
+    KRATOS_DEFINE_APPLICATION_VARIABLE( RANS_APPLICATION, std::vector<std::string>, ANALYSIS_STEPS )
```

Comment:
This is identify whether the given model part has gone through some of the steps, if not do those steps, if they are done skip those steps.

Example case is normal calculations. It is required by incompressible potential flow, vms_monolithic. So if someone uses incompressible potential flow to initialize velocity field, then normal calculation will be done there. With these ANALYSIS_STEPS, vms_monolithic knows normal calculation is alread y done so it can skip it., if they don't use incompressible potential flow for flow velocity initialization, then VMS_monolithic has to do normal calculation.

Reply:
I like the idea, but I think it would be better to consensuate this. My point is that any string value of these `ANALYSIS_STEPS` is now valid. It would be better to consensuate these (I'm thinking in a flag-like implementation).

Reply:
The problem with FLAGS is, it needs to be defined before hand (in few different places). Having strings doesn't have this difficulty but they are also risky (one can mis-spell), but it gives unlimited freedom. For the time being I'll keep it as strings untill we agree with common method for this in Kratos.